### PR TITLE
Lp 291 plp page template

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link

https://companieshouse.atlassian.net/browse/LP-291

### Change description

Update name page to selectively display lp or private lp content. 
Added private lp content.

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by lduranteau** [comment]
can you create a name subfolder and add all the name.njk pages to it?

**Comment by mattch1** [comment]
Will need to add the other two links for Acs 3 and 4.

File: src/views/name.njk [hunk 1]
```diff
@@ -7,160 +7,11 @@
...
-          }) }}
-          
-        </fieldset>
-      </div>
-
-      {% include "includes/save-and-continue-button.njk" %}
-    </form>
-  </div>
-</div>
+{% if props.parameters['which-type'] == "registerPflp" %}
```
[the remark this answers is not in the document]
Comment (mattch1) [reply src/views/name.njk]: Realigned the conditionals

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by lduranteau** [comment]
> > can you create a name subfolder and add all the name.njk pages to it?
> 
> Not sure we need this but added a subfolder, name.njk should stay where it is as this will disrupt the routing.

yes for name.njk I agree 
it was just to group the subpages in a single location

**Review by lduranteau — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Pull request merged**